app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/dashboard/DashboardScreen.kt

```diff
@@ -95,7 +95,33 @@ fun DashboardScreen(
                         fontWeight = FontWeight.Normal
                     )
                 }
-                items(25) { PastInterviewCard() }
+                items(25) {
+                    Column(
```

Comment:
This was supposed to be only for testing purpose not for committing here. PasInterViewCard() should be shown here.